internal/controller/nim/account_controller.go

```diff
@@ -69,6 +69,7 @@ var (
 func (r *AccountReconciler) SetupWithManager(mgr ctrl.Manager, ctx context.Context) error {
 	// TODO: Copied from original main.go... Should it be FromContext?
 	logger := ctrl.Log.WithName("controllers").WithName("ModelRegistryInferenceService")
```

Comment:
is this logger name correct?

Reply:
Probably not, but this was not introduced in this PR. This was introduced when the project migrated to kubebuilder v4. See https://github.com/opendatahub-io/odh-model-controller/pull/324.

I can fix it in this PR if you prefer or do it in a different PR. We can even include the TODO comment @Jooho was referring to here in the same PR. Both were introduced in the migration PR and not here.

Reply:
please udpate it.

Reply:
Resolved in https://github.com/opendatahub-io/odh-model-controller/pull/383/commits/835ba0b37ba8095adf1b2200bcaf990f0361491b.